### ISSUE TITLE
[BUG] : Crash on launch when developing with v1.5.0

### ISSUE THREAD

**Issue by Insprill** [Zagrios/bs-manager#450]
## Bug Description
When trying to run the program on the v1.5.0 branch (541a14d2a7e3257635c9eadb48a2eecc13467d40), it crashes immediately with the following exception.

```
18:57:32.629 › Unhandled Exception Error: App version is not a valid semver version: "0.0"
    at newError (/home/insprill/dev/misc/bs-manager/node_modules/builder-util-runtime/src/index.ts:55:17)
    at AppUpdater (/home/insprill/dev/misc/bs-manager/node_modules/electron-updater/src/AppUpdater.ts:232:21)
    at BaseUpdater (/home/insprill/dev/misc/bs-manager/node_modules/electron-updater/src/BaseUpdater.ts:11:5)
    at AppImageUpdater (/home/insprill/dev/misc/bs-manager/node_modules/electron-updater/src/AppImageUpdater.ts:15:5)
    at doLoadAutoUpdater (/home/insprill/dev/misc/bs-manager/node_modules/electron-updater/src/main.ts:32:20)
    at Object.get (/home/insprill/dev/misc/bs-manager/node_modules/electron-updater/src/main.ts:64:28)
    at AutoUpdaterService (/home/insprill/dev/misc/bs-manager/src/main/services/auto-updater.service.ts:19:9)
    at Function.getInstance (/home/insprill/dev/misc/bs-manager/src/main/services/auto-updater.service.ts:13:43)
    at <anonymous> (/home/insprill/dev/misc/bs-manager/src/main/ipcs/launcher-ipcs.ts:13:47)
    at IpcMainImpl.<anonymous> (/home/insprill/dev/misc/bs-manager/src/main/services/ipc.service.ts:38:13)
```

## Reproduction Steps
1. `git clone -b v1.5.0 https://github.com/Zagrios/bs-manager`
2. `cd bs-manager`
3. `npm i`
4. `npm run start`

## System Specs
OS: Arch Linux
Version: 541a14d2a7e3257635c9eadb48a2eecc13467d40

**Comment by Zagrios** [comment]
Oh yeah, I already faced this issue but didn't really take the time to look into it in more detail 😅 
I've been really busy lately, but I should be able to examine it more closely this week 👌

**Comment by Zagrios** [comment]
fix merged in v1.5.0 👌
